admin/workspace-providers/deployment.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Deployment
```

Comment:
For discoverability (especially with SEO) we should probably change this to "Workspace Provider Deployment"